tests/dynamic_data/base_test_by_used_servers.py

```diff
@@ -0,0 +1,23 @@
+import os
+import pytest
+
+from test_tools import external_db
+from tests.dynamic_data import base_test
+from tests.dynamic_data.publications import common_publications
+
+DIRECTORY = os.path.dirname(os.path.abspath(__file__))
+
+
+@pytest.mark.usefixtures('ensure_external_db')
+class TestSingleRestPublicationByUsedServers(base_test.TestSingleRestPublication):
+
+    @pytest.fixture(scope='class', autouse=True)
+    def class_fixture(self, request):
+        self.import_external_table(common_publications.INPUT_FILE_PATH, {
```

Comment:
To discuss

Reply:
Dynamically added fixtures?
https://stackoverflow.com/questions/46774636/how-to-dynamically-add-new-fixtures-to-a-test-based-on-the-fixture-signature-of